### DISCUSSION TITLE
LISTING_ITEMS : Vendors attempting to update existing items receive response to change product type, category, sub-category or item type keyword but the values are not present in the schema

### DISCUSSION THREAD

**Discussion by mcleigh86** [amzn/selling-partner-api-models#3309] [General]
We are seeing items that are existing receive a response telling us to adjust the product category, sub_category or item_type_keyword. However, the schema provided for that product type doesn't support the combination that the feedback is telling us to adjust. We're seeing these on existing items that were once originally listed with a specific category, sub_category and item_type_keyword that is no longer available to use, thus stopping the item from being updated.  

Is anyone else experiencing this? 

[EXCERCISE MAT Schema.docx](https://github.com/amzn/selling-partner-api-docs/files/8523636/EXCERCISE.MAT.Schema.docx)
[B07D18GYTT_request.docx](https://github.com/amzn/selling-partner-api-docs/files/8523639/B07D18GYTT_request.docx)
[B07D18GYTT_response.docx](https://github.com/amzn/selling-partner-api-docs/files/8523640/B07D18GYTT_response.docx)

**Comment by vaidas-lungis** [comment]
Can you please remove the docx and post the contents in plain text?

**Comment by chapmanjw** [comment]
@mcleigh86 The validation errors surrounding product_category, product_subcategory, and item_type_keyword are handled by the same mechanism that provides these validations on Vendor Central. At this time, the configuration for these values is manually configured. Contacting developer support (or having the vendor contact vendor support) with the missing details is currently the only mechanism available to get these updated.

We are working to address the gaps here in a more automated fashion, but I do not yet have timeframes as to when those improvements will be delivered. :-(
